lib/guides_style_18f/navigation.rb

```diff
@@ -1,14 +1,67 @@
 # @author Mike Bland (michael.bland@gsa.gov)
 
+require 'jekyll'
 require 'safe_yaml'
 
 module GuidesStyle18F
   module FrontMatter
     def self.load(basedir)
-      Dir[File.join basedir, 'pages', '**', '*.md'].map do |f|
-        [f[basedir.size + 1..-1], SafeYAML.load_file(f, safe: true)]
+      init_file_to_front_matter_map(basedir).merge(
+        site_file_to_front_matter(init_site(basedir)))
+    end
+
+    def self.init_site(basedir)
+      Dir.chdir(basedir) do
+        config = SafeYAML.load_file('_config.yml', safe: true)
+        adjust_config_paths(basedir, config)
+        site = Jekyll::Site.new(Jekyll.configuration(config))
+        site.reset
+        site.read
+        site
+      end
+    end
+    private_class_method :init_site
```

Comment:
No reason other than I never thought to go back and do it. Just made the change, which required moving a couple functions around.